src/plugins/vis_type_vega/public/vega_type.ts

```diff
@@ -43,10 +43,17 @@ export const createVegaTypeDefinition = (
     title: 'Vega',
     getInfoMessage,
     description: i18n.translate('visTypeVega.type.vegaDescription', {
-      defaultMessage: 'Create custom visualizations using Vega and Vega-Lite',
+      defaultMessage: 'Use Vega to create new types of visualizations.',
       description: 'Vega and Vega-Lite are product names and should not be translated',
     }),
+    note: i18n.translate('visTypeVega.type.vegaNote', {
+      defaultMessage: 'Requires programming knowledge.',
```

Comment:
I'd prefer if we could describe this differently. Vega is not a programming language, and there is zero knowledge about programming requried to write a Vega spec and vice versa knowledge about programming does not help you at all writing a vega specification. I think we should really just put it like that: "Requires knowledge about the Vega grammar."

cc @gchaps

Reply:
@stratoula I suggest this text:

Requires knowledge of Vega syntax.

Reply:
Thank you @gchaps 😄